### PR TITLE
Abandon 'License' service

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Feature Wiki Page: https://docu.ilias.de/goto_docu_wiki_wpage_5589_1357.html

### CODE REVIEW

File: setup/sql/5_4_hotfixes.php [hunk 1]
```diff
@@ -17,4 +17,19 @@
...
+?>
+<#4>
+<?php
+if ($ilDB->tableExists('license_data')) {
+	$ilDB->dropTable('license_data');
+}
+?>
+<#5>
+<?php
+$settings = new ilSetting('license');
```
Comment (smeyer-ilias) [inline setup/sql/5_4_hotfixes.php]: A "DELETE FROM settings WHERE module = ".$ilDB->quote('license', "text");
would reduce dependencies to non setup classes.

**Review by smeyer-ilias — COMMENTED** [review]
Only one note regarding the db update step.

File: setup/sql/5_4_hotfixes.php (hunk 1 above)
Comment (mjansenDatabay) [reply setup/sql/5_4_hotfixes.php]: @smeyer-ilias I will change this, thx!

**Comment by matthiaskunkel** [comment]
Jour Fixe, 17 DEC 2018 : We highly appreciate this pull request and accept the removal of the Licence service with 5.4 and for trunk.

**Pull request merged**